src/components/FiltersBar/FiltersBar.tsx

```diff
@@ -0,0 +1,501 @@
+        value={!!matureContentRatingFilter}
+      ></Checkbox>
+    </FilterContentContainer>
+  )
+
+  return (
+    <>
+      {betweenBaseAndSMMatch ? (
+        <MobileFilterDialog
+          onExitClick={() => setiIsFiltersOpen(false)}
+          showDialog={isFiltersOpen}
+          title="Filters"
+          content={
+            <>
+              <MobileFilterContainer>
+                <Text secondary variant="overhead">
+                  Sort by
+                </Text>
+                <Select
+                  size="small"
+                  helperText={null}
+                  value={sortVideosBy}
+                  items={SORT_OPTIONS}
+                  onChange={handleSorting}
+                />
+              </MobileFilterContainer>
+              <MobileFilterContainer>
+                <Text secondary variant="overhead">
+                  Date uploaded
+                </Text>
+                {dateUploadedInputs}
+              </MobileFilterContainer>
+              <MobileFilterContainer>
+                <Text secondary variant="overhead">
+                  Length
+                </Text>
+                {videoLengthInputs}
+              </MobileFilterContainer>
+              <MobileFilterContainer>
+                <Text secondary variant="overhead">
+                  License
+                </Text>
+                {licenseInputs}
+              </MobileFilterContainer>
+              <MobileFilterContainer>
+                <OtherFilterStyledText secondary variant="overhead">
+                  <OtherFilterStyledIcon />
+                  Exclude:
+                </OtherFilterStyledText>
+                {otherFiltersInputs}
+              </MobileFilterContainer>
+            </>
+          }
+          primaryButton={{
+            text: 'Apply',
+            onClick: handleApplyFilter,
+          }}
+          secondaryButton={{
+            text: 'Clear',
+            disabled: canClearAllFilters === false,
+            onClick: clearAllFilters,
+          }}
+        />
+      ) : (
+        <CSSTransition in={isFiltersOpen} timeout={100} classNames="filters" unmountOnExit>
+          <FiltersContainer open={true}>
+            <FiltersInnerContainer>
+              <PopoverDialog
+                content={dateUploadedInputs}
+                footer={
+                  <>
+                    <Button
+                      onClick={clearDateUploadedFilter}
+                      size="small"
+                      variant="secondary"
+                      disabled={dateUploadedFilter === undefined}
+                    >
+                      Clear
+                    </Button>
+                    <Button size="small" onClick={handleApplyFilter}>
+                      Apply
+                    </Button>
+                  </>
+                }
+              >
+                <Button badge={canClearDateUploadedFilter && 1} variant="secondary" onClick={() => null}>
```

Comment:
We shouldn't add `onClick={() => null}` on every Button here. I understand that you do this to add pointer styles but that's not the right way to go about this. It may be misleading to somebody not aware of why it's used. Maybe instead we can just add pointer styles to button regardless of whether it has `onClick` or not